### PR TITLE
Make "npm create nil-hardhat-template" work.

### PR DESCRIPTION
**Pull request by avm**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
```
Comment (ukorvl) [inline cli.js:1]: recommend to move cli.js into scripts folder in the root. it is kinda a typical pattern to store scripts in npm packages. but it is not super important right now.

**Review by Rolaman — APPROVED** [review]

File: cli.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        description: "A new project on the =nil; platform",
+        main: "index.js",
+        scripts: {
+            tests: "npx hardhat test --network nil",
+            lint: "biome check ignition/modules test",
+            "lint:fix": "biome check --write ignition/modules test"
+        },
+        dependencies: {},
+        devDependencies: {
+            "@nilfoundation/hardhat-plugin": "^0.15.0",
```
Comment (0xAleksaOpacic) [inline cli.js:27]: I don't like that we have a new entry where we need to maintain the versions. Usually i don't like "latest" but maybe it's worth here as it's just example project?

Comment (ukorvl) [reply cli.js:27]: Sasha is going to make the same pull request in nil repo.

**Review by ukorvl — CHANGES_REQUESTED** [review]
we should create this in nil repo, not here. here we can port the code from nil if needed

**Review by 0xAleksaOpacic — CHANGES_REQUESTED** [review]
Thank you for opening this PR, i think this is super cool thing to have in general. 

But i have a few concerns:
1. Why is this opened in `nil-hardhat-example` and not `nil` repository? 
2. Why is this in example project at all? For me it would make sense to publish separate npm package (similar to what react has https://www.npmjs.com/package/create-react-app) and then users can just do:
```
npm i create-hardhat-nil
```
and then use
```
npx create-nil-hardhat my-project
cd my-project
npm install
```

**Comment by avm** [comment]
> 2. Why is this in example project at all?

Right, probably a separate package makes more sense.

> users can just do:
> [4 steps]

Apparently npm create wraps all of those into 1 step (`npm create ...`).

**Pull request closed without merging**